input.js

```
@@ -0,0 +1,36 @@
+console.log("Start");
+console.log("Start");  // ❌ Repetido
+
+// 🔥 5. Logging en producción
+console.log("Este mensaje debería eliminarse en producción");
+
+// 🔥 6. Variable no usada
+const secret = "123456789";  // ❌ Declarada pero no usada
+
+// 🔥 Código vulnerable agregado a propósito para test de BugBot
+const userInput = process.argv[3];
+eval(userInput); // 🚨 Eval con input externo, muy inseguro
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)